bcmc/src/main/java/com/adyen/checkout/bcmc/DefaultBcmcDelegate.kt

```diff
@@ -38,6 +39,7 @@ internal class DefaultBcmcDelegate(
     private val publicKeyRepository: PublicKeyRepository,
     private val configuration: BcmcConfiguration,
     private val cardValidationMapper: CardValidationMapper,
+    private val cardEncrypter: DefaultCardEncrypter,
```

Comment:
Should this be `CardEncrypter` or `DefaultCardEncrypter`?

Reply:
Good catch!